src/db/z_db.erl

```diff
@@ -346,6 +357,7 @@ equery(Sql, Parameters, #context{} = Context) ->
 equery(Sql, #context{} = Context, Timeout) when is_integer(Timeout) ->
     equery(Sql, [], Context, Timeout).
 
+-spec equery(sql(), parameters(), #context{}, integer()) -> any().
```

Comment:
The spec in line 360 uses a record, please define a type for the record and use that instead.